src/common/local_file_system.cpp

```diff
@@ -439,7 +444,13 @@ unique_ptr<FileHandle> LocalFileSystem::OpenFile(const string &path_p, FileOpenF
 			}
 		}
 	}
-	return make_uniq<UnixFileHandle>(*this, path, fd, flags);
+
+	auto file_handle = make_uniq<UnixFileHandle>(*this, path, fd, flags);
+	if (opener) {
+		file_handle->logger = FileHandleLogger::CopyLoggerPtr(*opener);
+		DUCKDB_LOG_FILE_HANDLE_OPEN((*file_handle));
+	}
+	return file_handle;
```

Comment:
This part here causes the following conversion error
```
/home/runner/work/duckdb/duckdb/src/common/local_file_system.cpp:453:16: error: could not convert ‘file_handle’ from ‘unique_ptr<duckdb::UnixFileHandle,default_delete<duckdb::UnixFileHandle>,[...]>’ to ‘unique_ptr<duckdb::FileHandle,default_delete<duckdb::FileHandle>,[...]>’
  453 |         return file_handle;
      |                ^~~~~~~~~~~
      |                |
      |                unique_ptr<duckdb::UnixFileHandle,default_delete<duckdb::UnixFileHandle>,[...]>
```
see [here](https://github.com/duckdb/duckdb/actions/runs/14036823227/job/39306493489?pr=16758).

Reply:
ah thanks! yea this needs an `std::move`